docker-compose-ssl.yml

```diff
@@ -3,6 +3,7 @@ services:
   nginx-proxy:
```

Comment:
For bmsapp container, add the BMS_PASSWORD environment property

Reply:
done